static/css/reference.css

```diff
@@ -79,3 +79,25 @@
 .post-two-pane .post-content .main-content h4 {
   font-size: 18pt;
 }
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh - var(--header-height) + 50px);
+}
+
+.sidebar-header {
+  flex-shrink: 0;
+  border-bottom: 1px solid #eee;
+}
+
+.sidebar-content {
+  flex: 1;
+  overflow-y: auto;
+  scrollbar-width: thin;
+}
+
+.sidebar-content ul {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Can you please add this to `assets/css/extended/custom.css` as `.sidebar` is a global class, not only for references?

Also, the sidebar won't be hidden for smaller screens (ex. smartphones) anymore. You can find the media query in the same file.

https://github.com/coalton-lang/coalton-website/blob/42cad1db9fe24d16fd8f581cacdb071988575052/assets/css/extended/custom.css#L107